docker/webui/run_openqa.sh

```diff
@@ -1,15 +1,49 @@
 #!/bin/bash
 
-# prepare environment
-if [ ! -f /data/db/db.sqlite ]; then
-  /usr/share/openqa/script/initdb --user geekotest --init_database
-else
-  /usr/share/openqa/script/upgradedb --user geekotest --upgrade_database
-fi
+function waitForDBcreation() {
+  echo "Waiting for DB creation"
+  sleep 5
```

Comment:
I consider this bad programming style and don't want to promote that. I wonder what problem exactly you observed because maybe this is something we can even fix in the corresponding services that might struggle with this. Or you make it a bit smarter like:

```bash
while ! sudo -u geekotest psql --list | grep -q openqa; sleep .1; done
```

Reply:
I commented some ideas in the Martchus comment, but I am going to check this

Reply:
It works with minimal modifications, so I will use this